app/src/org/odk/collect/android/views/ODKView.java

```diff
@@ -453,21 +398,11 @@ public void setOnFocusChangeListener(OnFocusChangeListener l) {
         }
     }
 
-
-    /*
-     * (non-Javadoc)
-     * @see android.view.View.OnLongClickListener#onLongClick(android.view.View)
-     */
     @Override
     public boolean onLongClick(View v) {
         return false;
     }
     
-
-    /*
-     * (non-Javadoc)
-     * @see android.view.View#cancelLongPress()
-     */
```

Comment:
I'm going to say that these non-Javadoc comments are bad form.
They add clutter, can get out of date, and contain information that is easily looked up. A simple Override annotation and good method names should suffice.